velox/docs/functions/spark/datetime.rst

```diff
@@ -55,3 +55,7 @@ These functions support TIMESTAMP and DATE input types.
 .. spark:function:: year(x) -> integer
 
     Returns the year from ``x``.
+    
+.. spark:function:: add_months(startDate, numMonths) -> integer
+
+    Returns the date that is ``numMonths`` after ``startDate``.
```

Comment:
Would you clarify whether numMonths can be zero or negative and what happens on overflow?

Reply:
add doc for zero or negative.
for overflow, `long calcMonths = monthCount + monthsToAdd;  // safe overflow` seems no specific process in spark, just round. and Spark also does not document this behavior.

Reply:
I did notice that Spark documentation is lacking. However, Spark has a particular behavior that we need to match. Let's figure out what this behavior is, match it and document. CC: @rui-mo

Reply:
I couldn't agree more. 
check spark code： it does the check after arithmetic, I'll follow the same logic, and check overflow after `addToDate`.

Reply:
I couldn't agree more.
check spark code： it does the check after arithmetic, but velox `addToDate` seems to hide this issue, I'll try another method.

Reply:
from the code `https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/DateTimeImpl.h#L96` the `addToDate` does not process the overflow ( year be cast to short, month to int), seems can't get any error information out of `addToDate`?

Reply:
Consider using an alternative method instead of directly relying on the `addToDate`.

Reply:
@mbasmanova overflow is covered. Could you help to review this PR.